src/wp-admin/js/word-count.js

```diff
@@ -13,17 +41,19 @@
 
 		shortcodes = this.settings.l10n.shortcodes;
 
+		// If there are any localization shortcodes add this as type in the settings.
 		if ( shortcodes && shortcodes.length ) {
 			this.settings.shortcodesRegExp = new RegExp( '\\[\\/?(?:' + shortcodes.join( '|' ) + ')[^\\]]*?\\]', 'g' );
 		}
 	}
 
+	// Default settings.
 	WordCounter.prototype.settings = {
 		HTMLRegExp: /<\/?[a-z][^>]*?>/gi,
 		HTMLcommentRegExp: /<!--[\s\S]*?-->/g,
 		spaceRegExp: /&nbsp;|&#160;/gi,
 		HTMLEntityRegExp: /&\S+?;/g,
-		connectorRegExp: /--|\u2014/g,
+		connectorRegExp: /--|\u2014/g, // \u2014 = em-dash.
```

Comment:
In order to retain consistency, you might want to consider putting the comment above the property. see line 86 of this file.